tensorflow-core/tensorflow-core-api/src/gen/annotations/org/tensorflow/op/Ops.java

```diff
@@ -336,10 +337,10 @@
 
   public final SignalOps signal;
 
-  public final TrainOps train;
-
   public final QuantizationOps quantization;
 
+  public final TrainOps train;
```

Comment:
Looks like this got updated in after the shape change was integrated. @karllessard do you think it's worth making a separate PR (or just you push the ops update separately) so we don't have commits that make unintentional changes?

Reply:
I think it is up to you @JimClarke5 , normally this file should have been committed in your previous PR adding shape operators. Ideally we would commit it as a separate PR to keep this history clean, but I won't insist neither if you prefer we keep things simpler and just merge this PR as is.